Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -230,6 +230,14 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
 
         applyTheme()
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        if User.shared.showsWelcomeScreen {
+            present(WelcomeScreen(), animated: true)
```

Comment:
I suggest to add this logic to the BrowserViewController - presentIntroIfNeeded logic:

https://github.com/ecosia/ios-browser/blob/2abf9abf355e6950c8a2bf460f46465d78766faf/Client/Frontend/Browser/BrowserViewController.swift#L2017

So we have that all in one place